src/configuration/SSH/OpenSSH/7.3/sshd_config

```diff
@@ -0,0 +1,118 @@
+#-----------------------------
+# Daemon
+#-----------------------------
+
+# Specifies the port number that sshd(8) listens on.
+# The default is 22.
+#Port 1022
+
+# Specifies the local addresses sshd(8) should listen on.
+# The default is to listen on all local addresses.
+#ListenAddress 192.168.23.42
+
+# Specifies a file containing a private host key used by SSH.
+# The default is
+## /etc/ssh/ssh_host_dsa_key,
+## /etc/ssh/ssh_host_ecdsa_key,
+## /etc/ssh/ssh_host_ed25519_key and
+## /etc/ssh/ssh_host_rsa_key for protocol version 2.
+HostKey /etc/ssh/ssh_host_ed25519_key
+HostKey /etc/ssh/ssh_host_rsa_key
+
+# Specifies the maximum number of concurrent unauthenticated connections
+# to the SSH daemon.
+# The default is 10:30:100.
+MaxStartups 3
+
+# Specifies the maximum number of authentication attempts permitted per
+# connection.
+# The default is 6.
+MaxAuthTries 1
+
+# Specifies the maximum number of open shell, login or subsystem (e.g. sftp)
+# sessions permitted per network connection.
+# The default is 10.
+MaxSessions 6
+
+#-----------------------------
+# Authentication
+#-----------------------------
+
+# The server disconnects after this time if the user has not
+# successfully logged in.
+# The default is 120 seconds.
+LoginGraceTime 60
+
+# Specifies whether root can log in using ssh(1).
+# The default is "prohibit-password".
+PermitRootLogin no
+
+# Specifies the file that contains the public keys that can be used for
+# user authentication.
+# The default is ".ssh/authorized_keys .ssh/authorized_keys2".
+AuthorizedKeysFile .ssh/authorized_keys
+
+# Specifies whether password authentication is allowed.
+# The default is "yes".
+PasswordAuthentication no
```

Comment:
I don't disagree in theory. This is more of a practical issue. Many people rely on password-based logins. As I wrote above: It'll make far more sense to write a paragraph or two on the subject of password-less authentication for SSH than just to ship a configuration file. A simple idea would be to write about the benefits of password-less auth and include an example of how to change this line in the `sshd` configuration.

As a default without any proper explanation I'm personally opposed. But I think this is something that should be discussed on the mailing-list as there're probably very different opinions on the matter: https://lists.cert.at/cgi-bin/mailman/listinfo/ach